includes/class-coblocks-form.php

```diff
@@ -452,6 +426,12 @@ public function render_field_radio( $atts, $content ) { // phpcs:ignore Generic.
 
 		$this->render_field_label( $atts, $label );
 
+		if ( $atts['isInline'] ) {
```

Comment:
I am getting 
`Notice: Undefined index: isInline in /var/www/html/wp-content/plugins/coblocks/includes/class-coblocks-form.php on line 429` 
and 
`Notice: Undefined index: isInline in /var/www/html/wp-content/plugins/coblocks/includes/class-coblocks-form.php on line 448`


Should be able to resolve this by using `if ( isset( $atts['isInline'] ) )`

Reply:
Resolved in 1ebdccc